app/views/smart_answers/landing.html.erb

```diff
@@ -23,9 +23,14 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= render "govuk_publishing_components/components/title", {
-      title: start_node.title,
-    } %>
+    <div class="govuk-!-margin-top-8">
```

Comment:
Could the `margin-top` go on the column itself, rather than adding an extra element?